scripts/commands/immortal.lua

```diff
@@ -0,0 +1,69 @@
+---------------------------------------------------------------------------------------------------
+-- func: immortal <player>
+-- desc: Sets a target to be unkillable
+---------------------------------------------------------------------------------------------------
+
+cmdprops =
+{
+    permission = 1,
+    parameters = "s"
+}
+
+function error(player, msg)
+    player:PrintToPlayer(msg)
+    player:PrintToPlayer("!immortal {player}")
+end
+
+function onTrigger(player, hp, target)
+    -- validate target
+    local targ
+    local cursor_target = player:getCursorTarget()
+
+    if target then
+        targ = GetPlayerByName(target)
+        if not targ then
+            error(player, string.format( "Player named '%s' not found!", target ) )
+            return
+        end
+    elseif cursor_target and not cursor_target:isNPC() then
+        targ = cursor_target
+    else
+        targ = player
+    end
+
+    if not targ:isNPC() then
+        if targ:isAlive() then
+      
+            local immortal = false
+            if targ:isPC() then
+                if targ:getCharVar("Immortal") == 1 then
+                    targ:delStatusEffect(tpz.effect.ERGON_MIGHT)
```

Comment:
um didn't catch the use of a status effect my first look over. lets not do that. 

Lets just used addStatusEX to make its a duplicate of brew effect instead, for the lols :+1: